### DISCUSSION TITLE
faker.image.dataUri() -> Buffer is not defined

### DISCUSSION THREAD

**Discussion by Tsyklop** [faker-js/faker#3143] [Q&A] [answered]
Got error `Buffer is not defined` when call `faker.image.dataUri()`. What wrong? Faker 9.0.2, React. NodeJs 22

**Comment by matthewmayer** [comment, marked as answer]
https://stackoverflow.com/questions/67804053/uncaught-referenceerror-buffer-is-not-defined-in-react

**Reply by ST-DDT** [reply]
Should we add it to the restricted globals in our lint config?

````ts
import globals from 'globals';

const browserGlobals = new Set(Object.keys(globals.browser));
const disallowedGlobals = Object.keys(globals.node).filter(
  (e) => !browserGlobals.has(e)
);


{
  files: ['src/**/*.ts'],
  rules: {
    'no-restricted-globals': ['error', 'Intl', ...disallowedGlobals],
  },
},
````

Or is there a better way to do that?

**Reply by ST-DDT** [reply]
Here is the fix: #3144

- #3144

**Comment by ST-DDT** [comment]
As a workaround use:

````ts
faker.image.svg({type: 'svg-uri'});
````

Or similar.
